openquake/hazardlib/gsim/gmpe_table.py

```diff
@@ -302,50 +301,24 @@ class GMPETable(GMPE):
 
     GMPE_TABLE = None
 
-    def __init__(self, gmpe_table=None):
-        """
-        If the path to the GMPE table is not assigned as an attribute of the
-        class then instantiate with a path to the GMPE table. If the
-        gmpe_table is input this will replace the path to the hdf5 file if it
-        has already been assigned. Otherwise, if the path to the hdf5 file
-        is not assigned as a property or the class nor is it input at
-        instantiation, an error will be raised
-
-        :param str gmpe_table:
-            Path to the hdf5 file containing the GMPE table
-        """
-        if not self.GMPE_TABLE:
-            if gmpe_table:
-                if os.path.isabs(gmpe_table):
-                    self.GMPE_TABLE = gmpe_table
-                else:
-                    # NB: (hackish) GMPE_DIR must be set externally
-                    self.GMPE_TABLE = os.path.abspath(
-                        os.path.join(self.GMPE_DIR, gmpe_table))
-            else:
-                raise IOError("GMPE Table Not Defined!")
-        super().__init__(gmpe_table=gmpe_table)
-        self.imls = None
-        self.stddevs = {}
-        self.m_w = None
-        self.distances = None
-        self.distance_type = None
-        self.amplification = None
-        # NB: it must be possible to instantiate a GMPETable even if the
-        # the .hdf5 file (GMPE_TABLE) does not exist; the reason is that
-        # we want to run a calculation on machine 1, copy the datastore
-        # on machine 2 (that misses the .hdf5 files) and still be able to
-        # export the results, i.e. to instantiate the GsimLogicTree object
-        # which is required by the engine to determine the realizations
-        if os.path.exists(self.GMPE_TABLE):
-            with h5py.File(self.GMPE_TABLE, "r") as f:
-                self.init(f)
-
-    def init(self, fle):
+    amplification = None
```

Comment:
Can you pls add a comment to explain the meaning of this variable?

Reply:
Will do in the next PR